staff/acct/sorry/sorry

```diff
@@ -32,6 +36,15 @@ if [ -z "$(getent passwd "$sorriedUser")" ]; then
 fi
 
 sorryFile=$2
+
+if [[ ! -f "$sorryFile" ]]; then
```

Comment:
Would recommend using single brackets here in line with what we have in the rest of the script